guides/source/routing.md

```diff
@@ -127,20 +143,26 @@ creates seven different routes in your application, all mapping to the `Photos`
 | PATCH/PUT | /photos/:id      | photos#update     | update a specific photo                      |
 | DELETE    | /photos/:id      | photos#destroy    | delete a specific photo                      |
 
-NOTE: Because the router uses the HTTP verb and URL to match inbound requests, four URLs map to seven different actions.
+Because the router uses the HTTP verb *and* path to match inbound requests, four URLs map to seven different controller actions. For example, the same `photos/` path matches to `photos#index` when the verb is `GET` and `photos#create` when the verb is `POST`.
```

Comment:
Starting a sentence with Because (while grammatically correct) [feels similar to But ](https://3.basecamp.com/3076981/buckets/35498807/todos/7116145100#__recording_7387347313). I'm not sure if we're avoiding it as well.